README.md

```diff
@@ -349,8 +350,8 @@ module.exports = require("babel-loader").custom(babel => {
         plugins: [
           ...(cfg.options.plugins || []),
 
-          // Include a custom plugin in the options.
-          myPlugin,
+          // Include a custom plugin in the options and passing it the customOptions object.
+          (...pluginArgs) => myPlugin(customOptions, ...pluginArgs),
```

Comment:
This is not how you would pass option to a Babel plugin, it should be something like this:
```suggestion
          [myPlugin, customOptions],
```